website/client/src/components/groups/group.vue

```diff
@@ -10,7 +10,7 @@
     <group-gems-modal />
     <div class="col-12 col-sm-8 standard-page">
       <div class="row">
-        <div class="col-12 col-md-6 title-details">
+        <div class="col-12 col-md-4 title-details">
```

Comment:
I think changing this to md-4 is causing some issues

<img width="1024" alt="Schermata 2020-10-07 alle 15 16 34" src="https://user-images.githubusercontent.com/589911/95335966-56e44980-08b0-11eb-8620-f2196a1392f6.png">

The title should have more space to expand

Reply:
It should be the other way around md-8 for the title and md-4 for the badges

Reply:
Ok, want to make sure while I'm here that the spacing looks ok?

![Screenshot from 2020-10-07 18-44-34](https://user-images.githubusercontent.com/17064002/95367911-45f60100-08cd-11eb-8891-6a211aeca2b5.png)

They also stack on certain smaller sized screens since they can't shrink below 120 pixels, is this acceptable?

![Screenshot from 2020-10-07 18-48-50](https://user-images.githubusercontent.com/17064002/95368244-c4eb3980-08cd-11eb-808c-9f5b88838908.png)

For reference the same size screen looked like this before:

![Screenshot from 2020-10-07 18-47-24](https://user-images.githubusercontent.com/17064002/95368136-966d5e80-08cd-11eb-9855-041309cf346a.png)

Reply:
I haven't checked but maybe having both col-md-6 instead of 8 and 4 could result in a better support for smaller screens while still allowing longer titles

Reply:
Looks about the same with that. For reference the stacking occurs from widths of 768px to 892px if both are at col-md-6.

Reply:
Let's keep the two col-md-6 then